<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_Menu</class>
 <widget class="QWidget" name="Win_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4,12">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>326</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutMain" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonNewGame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
	padding-left:0px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
         </property>
         <property name="text">
          <string>Commencer la partie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLoadGame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
         </property>
         <property name="text">
          <string>Charger une partie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonParameters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
         </property>
         <property name="text">
          <string>Param√®tres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonQuit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:#E13c3c3c;
	color: #EEEEEE;
	margin-top:2px;
	margin-bottom:2px;
	margin-right:5px;
	margin-left:5px;
}

QPushButton::hover{
	background-color:#E14c4c4c;
	color: #FFFFFF;
	margin:0px;
}</string>
         </property>
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layoutSave">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="buttonRemoveSave">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>6</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>326</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
